### PR TITLE
Improves code coverage of Evaluators

### PR DESCRIPTION
**Pull request by Tpt**

### CODE REVIEW

File: lib/aggregators/Max.ts [hunk 1]
```diff
@@ -7,10 +7,7 @@ interface IExtremeState {
 export class Max extends BaseAggregator<IExtremeState> {
   public init(start: RDF.Term): IExtremeState {
     const { value } = this.extractValue(start);
-    if (start.termType === 'Literal') {
```
Comment (Tpt) [inline lib/aggregators/Max.ts:10]: `extractValue` already ensures that `start` is a literal.

File: lib/evaluators/evaluatorHelpers/AsyncRecursiveEvaluator.ts [hunk 2]
```diff
@@ -97,7 +97,7 @@ export class AsyncRecursiveEvaluator extends BaseExpressionEvaluator
   // TODO: Remove?
   private async evalAggregate(expr: E.Aggregate, _mapping: RDF.Bindings): Promise<E.Term> {
     if (!this.context.aggregate) {
-      throw new Err.NoExistenceHook();
```
Comment (Tpt) [inline lib/evaluators/evaluatorHelpers/AsyncRecursiveEvaluator.ts:100]: This is not the correct error.

File: lib/aggregators/Max.ts (hunk 1 above)
Comment (rubensworks) [reply lib/aggregators/Max.ts:10]: There might be some TS magic we can apply here to automatically let `start` be inferred as Literal.
Something like `start a RDF.Iteral` might be appendable to the return type of `extractValue` (but I don't see this as in-scope for this bounty, just optional :-) ).

**Review by rubensworks — APPROVED** [review]
🎉

File: lib/aggregators/Max.ts (hunk 1 above)
Comment (rubensworks) [reply lib/aggregators/Max.ts:10]: Also sounds good to me :-)

**Pull request merged**